package.json

```diff
@@ -109,7 +109,7 @@
     "rimraf": "^3.0.2",
     "rollup": "^2.32.1",
     "rollup-plugin-terser": "^7.0.2",
-    "ts-jest": "^27.0.3",
+    "ts-jest": "26.5.6",
```

Comment:
Why is this being downgraded?

Reply:
When I was using ts-jest@27, I encountered some [problems]( https://github.com/kulshekhar/ts-jest/issues/2611#issuecomment-851014389)
Then I upgraded JEST to version 27 and encountered this [problem](https://githubmemory.com/repo/facebook/fbjs/issues/441) again
In the end, I can only use version 26 first

Reply:
In addition,I will fix the problem left over from converting the TS file, But this may take some time, as strict type checking may require a lot of additional fixes:cold_sweat: